CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+* New Api module which exposes (via the `ApiFacade`) the internal functions of phel for PHP (#551)
```

Comment:
New Api module which exposes (via the `ApiFacade`) the functions documentation of Phel (#551)